python/ray/autoscaler/v2/instance_manager/common.py

```diff
@@ -0,0 +1,272 @@
+import time
+from dataclasses import dataclass
+from typing import Dict, List, Optional, Set
+
+from ray.core.generated.instance_manager_pb2 import Instance
+
+
+@dataclass
+class InvalidInstanceStatusError(ValueError):
+    """Raised when an instance has an invalid status."""
+
+    # The instance id.
+    instance_id: str
```

Comment:
this is instance manager instance id right?

Reply:
yes - will update name.